infrastructure/modules/aws/eks/patch-coredns.tf

```diff
@@ -2,9 +2,6 @@
 resource "null_resource" "patch_coredns" {
   provisioner "local-exec" {
     command = "kubectl -n kube-system patch deployment coredns --patch \"${yamlencode(local.patch_coredns_spec)}\""
-    environment = {
-      KUBECONFIG = module.eks.kubeconfig_filename
```

Comment:
Why remove this? I think deployment must work even if we do not update the `~/.kube/config` file

Reply:
It has been removed in v18 https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/UPGRADE-18.0.md look for : kubeconfig_filename